src/import/sst_service.rs

```diff
@@ -1234,6 +1243,37 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
     }
 }
 
+fn check_rewrite_rules_ordered(rewrite_rules: &[RewriteRule]) -> Result<()> {
+    let first_rewrite_rule =
+        rewrite_rules
+            .first()
+            .ok_or(sst_importer::Error::WrongRewriteRules(
+                "rewrite rules is empty".into(),
+            ))?;
+    let mut last_old_prefix = first_rewrite_rule.get_old_key_prefix();
+    let mut last_new_prefix = first_rewrite_rule.get_new_key_prefix();
+    for rewrite_rule in rewrite_rules {
```

Comment:
looks like we have an assumption here that if oldTableID1 < oldTableID2, then newTableID1 < newTableID2, do we have guarantee that it's always true?